3.10/aql/operations-replace.md

```diff
@@ -25,104 +25,171 @@ REPLACE <em>keyExpression</em> WITH <em>document</em> IN <em>collection</em></co
 Both variants can optionally end with an `OPTIONS { … }` clause.
 
 `collection` must contain the name of the collection in which the documents should
-be replaced. `document` is the replacement document. When using the first syntax, `document` 
-must also contain the `_key` attribute to identify the document to be replaced. 
+be replaced. `document` must be an object that contains the attributes and values 
+to set. All existing attributes in the stored document are removed from it and
+only the provided attributes are set (excluding system attributes).
+
+### `REPLACE <document> IN <collection>`
+
+Using the first syntax, the `document` object must have a `_key` attribute with
+the document key. The existing document with this key is replaced with the
+attributes provided by the `document` object.
```

Comment:
Again, except for the system properties. (What if the user reads only this paragraph?)